vault/README.md

```diff
@@ -12,60 +12,66 @@ The Vault check is included in the [Datadog Agent][2] package.
 
 #### Prerequisites
 
+This OpenMetrics-based integration has a latest version (`use_openmetrics`: true) and a legacy version (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][22].
+
 1. Ensure you have enabled [Prometheus metrics in the Vault configuration][3].
 
-2. For the Vault check to work properly, you need to either enable unauthenticated access to Vault metrics (Vault 1.3.0+) or provide a Vault client token:
+   * [Metrics starting with `vault.replication.fetchRemoteKeys`, `vault.replication.merkleDiff`, and `vault.replication.merkleSync`](#data-collected) are not reported unless the replication is in an unhealthy state. 
```

Comment:
I'm unsure of the placement of this paragraph. It does look like it belongs close to where the collected metrics are listed.

Reply:
```suggestion
```